src/js/structs/Node.js

```diff
@@ -39,8 +39,13 @@ class Node extends Item {
   }
 
   getUsageStats(resource) {
-    const total = this.get("resources")[resource];
-    const value = this.get("used_resources")[resource];
+    const total = (this.get("resources") || {})[resource];
```

Comment:
I wouldn't want to block this PR for this, but I think having a function like this to use is already a pattern for us:

```js
getResources() {
	return this.get("resources") || {};
}
```

Reply:
Yes. Orlando raised it in the original PR, my point was not adding something to the public API of a class that is not being used somewhere else. As in a like to carefully think what methods I want it to expose and not, and right now there is no requirement for that method.

Although for a struct it might not be as concerned as for a reacr component. We went without doing it on the original PR